.github/workflows/docker-node.yml

```diff
@@ -0,0 +1,52 @@
+name: Build Docker Images
+
+on:
+  push:
+    paths:
```

Comment:
Nice to see the solution that the workflow is only triggered on change of these specific files!

One thing I am wondering is if there is a way to only run the individual matrix jobs for the specific file which has changed so we don't update docker images unnecessarily, something like 
```
if contains(matrix.tag,  events.paths)
```
. I did some quick search, but it seems we would have to implement this manually which is probably not worth the effort at this point.

Reply:
that would be great, maybe there is a machanism from docker itself could to figure out if an image needs updating

Reply:
I was going with myself back-and-forth on when should this run and how. I think there is some benefit in knowing all images used for build/test came from same "batch". We'll see how it goes...